packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -111,79 +89,87 @@ export class ProjectService {
         return project;
     }
 
-    async create(user: SessionUser, data: CreateProject): Promise<Project> {
+    async create(
+        user: SessionUser,
+        data: CreateProject,
+    ): Promise<{ jobUuid: string }> {
         if (user.ability.cannot('create', 'Project')) {
             throw new ForbiddenError();
         }
 
-        const jobUuid = await this.startJob(undefined);
+        const job: CreateJob = {
+            jobUuid: uuidv4(),
+            jobType: JobType.CREATE_PROJECT,
+            jobStatus: JobStatusType.STARTED,
+            projectUuid: undefined,
+            steps: [
+                { stepType: JobStepType.TESTING_ADAPTOR },
+                { stepType: JobStepType.COMPILING },
+                { stepType: JobStepType.CREATING_PROJECT },
+            ],
+        };
 
-        const project = await this.createProject(user, data, jobUuid);
+        const doAsyncWork = async () => {
+            await this.jobModel.update(job.jobUuid, {
+                jobStatus: JobStatusType.RUNNING,
```

Comment:
don't we update `job` inside `tryjobstep` ? (with updated_at and jobStatus)

Reply:
We only update job steps in `tryJobStep`

Reply:
Let's move this call inside the try block.